src/lib.rs

```diff
@@ -1,8 +1,14 @@
 #![cfg_attr(feature = "nightly", feature(attr_literals, repr_align))]
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(feature = "std")]
+extern crate std as core;
```

Comment:
I wonder if why `std` should be renamed.

Reply:
Hmm good point, I could just use `extern crate core` directly here.